hypervisor/arch/x86/mmu.c

```diff
@@ -306,7 +306,8 @@ void init_paging(void)
 #ifdef CONFIG_DMAR_PARSE_ENABLED
 	for (i = 0U; i < entries_count; i++) {
 		entry = p_e820 + i;
-		if (entry->type == E820_TYPE_ACPI_RECLAIM) {
+		if ((entry->type == E820_TYPE_ACPI_RECLAIM) ||
+				(entry->type == E820_TYPE_RESERVED)) {
```

Comment:
Can be in one line.

BTW, what platform uses RESERVED ?  Do we know why?

Reply:
We have noticed it in MRB and KBL NUC